src/en/space-station-14/roleplay-lore/proposals/Departmental-Operating-Procedure.md

```diff
@@ -0,0 +1,122 @@
+# Departmental Operating Procedure (DOP)
+
+## Abstract
+
+--- 
+
+The DOP is an in-character set of guidelines oriented towards MRP servers. Each department will have it's own 'Operating Procedure' that will prepare it for any encounters it may face while in space. This will serve as a way to encourage roleplay and more belivable in-character behavior. 
+
+The DOP will act as a guideline for departments to follow but not rules. This will allow for chaos to still ensue even when everything goes according to plan. 
+
+## Core Idea
+
+My intention with a DOP is to encourage Roleplay but not to enforce it with server rules, this instead is a set of in-character books and guidebook entries that will push players into a more RP centric enviorment. This proposal is more focused towards MRP servers but these guidelines can still be found and used within LRP servers. 
+
+I will split these procedures between two categories 
+
+* Flavor Text
+  - Nanotrasen themed guides that offer more fun than actual advice, think of things like "Interogation 101" which will feature an interogation guide complete with responses to certain answers, pretty much useless in terms of advice but is a fun bit of lore that is distrubted to security teams across Nanotrasen stations
+* Guidelines
+    - Think of these as the original SOP, a set of straight forward points and guides to explain how a department is to be run, but do not limit roleplay and instead encourage more. With things like medical evaluations and alert procedures. These could be appended with 'guideline' or just left as is such as "Alert Procedure" instead of "Alert Procedure Guidelines". 
```

Comment:
Same as above. Guidelines, but not enforced? Then which mechanic prevents people from ignoring it?

Reply:
I don't believe anything prevents people from ignoring it. In it's current form this is largely a self-policed "standarized" roleplay. 

This is much like the last issue where I would like enforcment of some guidelinese but I feel it would have to be limited to MRP or jobs like command and sec (which I am trying to avoid as I don't enjoy an operating procedure that only covers two departments).

I will try to mention MRP ammendments or something similar to certain guidelines